script/publish-docs

```diff
@@ -31,8 +31,8 @@ esac
 echo "Submitting metadata envelopes to ${CONTENT_STORE_URL}."
 
 exec docker run --rm \
-  -e CONTENT_STORE_URL=${CONTENT_STORE_URL} \
-  -e CONTENT_STORE_APIKEY=${CONTENT_STORE_APIKEY} \
+  -e CONTENT_STORE_URL="${CONTENT_STORE_URL}" \
+  -e CONTENT_STORE_APIKEY="${CONTENT_STORE_APIKEY}" \
   -e TRAVIS_PULL_REQUEST="false" \
-  -v ${ROOT}/docs:/usr/control-repo \
+  -v "${ROOT}/docs:/usr/control-repo" \
```

Comment:
This was recently changed in deconst and should be "content-repo". That's likely causing the merge conflict.

s/control-repo/content-repo